src/Components/Searchbar/Searchbar.scss

```diff
@@ -3,6 +3,7 @@
   width: 100%;
   border: none;
   display: flex;
+  flex-flow: row nowrap;
```

Comment:
I made some changes to the css so that the search bar was all inline.

Reply:
Thanks for doing that! Not sure what happened to split it up but it look great